sdk/storage/Azure.Storage.Common/src/Azure.Storage.Common.csproj

```diff
@@ -16,6 +16,7 @@
       Microsoft Azure Storage quickstarts and tutorials - https://docs.microsoft.com/en-us/azure/storage/
       Microsoft Azure Storage REST API Reference - https://docs.microsoft.com/en-us/rest/api/storageservices/
     </Description>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
We should probably be adding this code to the ignore list for compiling common, instead explicitly compiling it into Blobs and Queues, since that is where it will end up.

We can probably also compile it into Common.Tests if need be.